site/gatsby-site/src/server/generated/graphql.ts

```diff
@@ -0,0 +1,280 @@
+import { GraphQLResolveInfo, GraphQLScalarType, GraphQLScalarTypeConfig } from 'graphql';
```

Comment:
This is the auto-generated typescript file that reflects the graphql schema. We can use it for internal development, which will be helpful for future API consumers.

Reply:
TODO: Instructions on how to curate the generated files.